### PR TITLE
[T4A5][W13-B1] Lim Hui Qi

### PR DESCRIPTION
**Pull request by LuMiN0uSaRc**
Use association classes

### CODE REVIEW

**Comment by LuMiN0uSaRc** [comment]
Ready to review

File: src/seedu/addressbook/data/person/Printer.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package seedu.addressbook.data.person;
+
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Printer.java:2]: please note, we will not review your PRs the next time if you don't submit clean PRs.
Avoid accumulating unrelated changes into a single PR. 
Tip: If you are currently on `branch1` and want to create a new `branch2`, you need to switch back to `master` branch before creating `branch2`. If not, changes in `branch1` will appear in the PR created from `branch2` later.

File: src/seedu/addressbook/data/person/Tagging.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
+package seedu.addressbook.data;
+
+import java.util.ArrayList;
+
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Tagging.java:4]: Please include a header comment to indicate what this interface/class/method does?

File: src/seedu/addressbook/data/person/Tagging.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+public class Tagging {
+    
+    private static ArrayList<String> listOfTaggings;
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Tagging.java:7]: recollect the discussion during tutorial on why this is not the right place to maintain the tagging list.
please check and differentiate the 2 diagrams in the activity description for better understanding

File: src/seedu/addressbook/data/person/Tagging.java [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    
+    private boolean isAdd;
+    private String personName;
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Tagging.java:10]: assume the application allows to add multiple people with same name;
now, how will you associate the tag to the right person?

**Comment by okkhoy** [comment]
@LuMiN0uSaRc 
Some comments added. Please ack & close the PR after reading comments.

File: src/seedu/addressbook/data/person/Tagging.java (hunk 4 above)
Comment (LuMiN0uSaRc) [reply src/seedu/addressbook/data/person/Tagging.java:10]: I will tag the right person with important information such as contact number/email as no multiple people can have the same contact number and email.

**Comment by LuMiN0uSaRc** [comment]
Noted with thanks!

**Pull request closed without merging**